### PR TITLE
IDENTITY-4670 - Dashboard single logout issue fix

### PR DESCRIPTION
**Pull request by ayomawdb**

### CODE REVIEW

File: modules/jaggery-apps/user-dashboard/dashboard/acs.jag [hunk 1]
```diff
@@ -121,5 +126,11 @@ if (samlResponse != null) {
     log.debug('store Session Id :::' + jSessionId);
 
     session.invalidate();
+
+    var sessionObject = application.get('sso_session_objects')[index];
+    if(sessionObject) {
```
Comment (darshanasbg) [inline modules/jaggery-apps/user-dashboard/dashboard/acs.jag]: formatting issues..!

Comment (ayomawdb) [inline modules/jaggery-apps/user-dashboard/dashboard/acs.jag]: Was already fixed with 843990216cf8d7012f78504346ac6c5f6508c2ee ("Format correction")

Comment (ayomawdb) [inline modules/jaggery-apps/user-dashboard/dashboard/acs.jag]: Sorry.. Agreed.

Comment (ayomawdb) [inline modules/jaggery-apps/user-dashboard/dashboard/acs.jag]: Fixed!

**Pull request merged**